### DISCUSSION TITLE
🚀Github Discussion 을 이용한 팀원 간의 소통 제안

### DISCUSSION THREAD

**Discussion by peanudge** [InnerCircleA/design-system#1] [Ideas]
저희가 하는 논의나 아이디어들이 휘발되지않고 남아있으면 좋을 것 같아서 Discussion을 적극적으로 활용해보고 싶습니다.

다만, Issues 와는 다른 용도로 좀더 가벼운 아이디어 제안, 혹은 학습에 도움이 되는 자료, 링크와 의견를 마구마구 올려주시면 좋을것 같아요.
😁😁

관련해서 의견 주시면 감사하겠습니다.

@gyutae100

**Comment by gyutae100** [comment]
저의 의견은 **매우 찬성** 입니다! 
굉장히 좋은 제안 감사합니다.

찬성하는 이유는, 
`휘발성` 측면 외 추가로, 자료의 `응집력`과 `연관성(?)` 측면에서 *최대한 깃허브 내 기능을 활용*하는 방향으로 커뮤니케이션 한다면, 협업 시 많은 이점이 있을 것으로 판단됩니다.

**깃 허브 내 효율적인 도구가 있다면, 언제든지 제안해주시면 환영**입니다.
*추후 필요하다면, 깃 허브 외의 도구 또한 활용*해보는 것도 좋겠다는 생각도 듭니다.

감사합니다.
이상입니다.
